React/src/components/FormControl/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { useForm } from 'react-hook-form';
+
+import { RegisterUser } from 'interfaces/RegisterUser';
+
+import styles from './styles.module.scss';
+
+export function FormControl() {
+  const { register, handleSubmit } = useForm<RegisterUser>();
+
+  return (
+    <div className={styles.pageContainer}>
+      <div className={styles.loginContainer}>
+        <form
+          onSubmit={handleSubmit((data) => console.log({ users: data }))}
+          className={styles.inputsContainer}
+        >
+          <h2 className={styles.dataTitle}>Nombre</h2>
+          <input {...register('name')} className={styles.dataInput} type="text" />
+
+          <h2 className={styles.dataTitle}>Apellido</h2>
+          <input {...register('lastName')} className={styles.dataInput} type="text" />
+
+          <h2 className={styles.dataTitle}>Email</h2>
+          <input {...register('email')} className={styles.dataInput} type="email" />
+
+          <h2 className={styles.dataTitle}>Password</h2>
+          <input {...register('password')} className={styles.dataInput} type="password" />
+
+          <h2 className={styles.dataTitle}>Confirmación De Password</h2>
+          <input {...register('passwordConfirmation')} className={styles.dataInput} type="password" />
+
+          <button type="submit" className={styles.loginButton}>
+            <h4 className={styles.loginTitle}>Login</h4>
```

Comment:
This `<h4>` is not needed either you can:
1. pass the content directly to the `<button>` (and move the styles applied on "loginTitle" to "loginButton")
2. use a `<span>` instead.

I would recommend the 1st option